storage/innobase/os/os0file.cc

```diff
@@ -152,6 +157,22 @@ class io_slots
 static io_slots *read_slots;
 static io_slots *write_slots;
 
+/**
+  Statistics for asynchronous I/O
+  @param[in] op operation type (aio_opcode::AIO_PREAD or aio_opcode::AIO_PWRITE)
+  @param[in] stats pointer to the structure to fill
```

Comment:
Can we please remove the misleading `[in]`? The second parameter is actually for output, not input.